src/cdk-experimental/scrolling/auto-size-virtual-scroll.ts

```diff
@@ -126,23 +147,130 @@ export class AutoSizeVirtualScrollStrategy implements VirtualScrollStrategy {
     this._addBufferPx = addBufferPx;
   }
 
+  /** Update the rendered content after the user scrolls. */
+  private _updateRenderedContentAfterScroll() {
```

Comment:
This method is getting pretty big, can it be broken into smaller ones?

Reply:
I still need to add some more logic here for calculating the amount of skew between what our item size estimate says our scroll position should be and what our scroll position actually is. Once I'm done with that I'll try to figure out how to best separate it.